README.md

```diff
@@ -1 +1,37 @@
-# spec
+# OpenEvents
+
+Events are everywhere.  However, event publishers tend to describe events differently.
+
+The lack of a common way of describing events means developers must constantly re-learn how to receive events.  This also limits the potential for libraries, tooling and infrastructure to aide the delivery of event data across environments, like SDKs, event routers or tracing systems.  The portability and productivity we can achieve from event data is hindered overall.
+
+Enter OpenEvents, a specification for describing event data in a common way.  OpenEvents seeks to ease event declaration and delivery across services, platforms and beyond.
+
+OpenEvents is a new effort and it's still under active development.  However, its working group has received a surprising amount of industry interest, ranging from major cloud providers to popular SaaS companies.  Our end goal is to offer this specification to the [Cloud Native Computing Foundation](https://www.cncf.io/).
+
```

Comment:
I think there's an AI to add a "usecases" doc. Would it make sense to add at least a placeholder doc and href into this PR so people can then incrementally add to the doc over time?

Reply:
The use-cases are in another PR.  Let me know if you think we should still link to it.